test/runnable/test42.d

```diff
@@ -6157,10 +6158,13 @@ void test11472()
     assert(x11472 == 10);
 }
 
+}
+
 /***************************************************/
 
 int main()
 {
+/+
```

Comment:
No you don't.

Reply:
Haha, I was unable to reproduce the failure on my machine. I'm doing a binary search here to find it on the machine that does fail. So no worries.